packaging/msi/SumoLogic.wixext/SumoLogicTests/ConfigUpdaterTests.cs

```diff
@@ -29,6 +29,17 @@ public void InstallationTokenAssertions(Config config, StreamReader sr)
             Assert.AreEqual(YamlNodeType.Scalar, sumologic.Children["installation_token"].NodeType);
             Assert.AreEqual(config.InstallationToken, sumologic.Children["installation_token"].ToString());
 
+            if (config.Ephemeral)

```

Comment:
what is the Ephemeral config option used for?

Reply:
oh nvm, got it - https://github.com/SumoLogic/sumologic-otel-collector/blob/11f9ba20816ccc17999d4084466d7bfdcdc3b364/pkg/extension/sumologicextension/README.md?plain=1#L60

Reply:
When set it automatically deletes the collector from the console after 12 hours of offline time.

![image](https://github.com/SumoLogic/sumologic-otel-collector/assets/1001835/e71a6697-cc04-47fa-934f-35b12b1b50c9)